src/fragments/lib/graphqlapi/js/optimistic-ui.mdx

```diff
@@ -158,19 +158,19 @@ const createMutation = useMutation({
     return newRealEstateProperty;
   },
   // When mutate is called:
-  onMutate: async (newRealEstateProperty: CreateRealEstatePropertyInput) => {
+  onMutate: async (newRealEstateProperty) => {
     // Cancel any outgoing refetches
     // (so they don't overwrite our optimistic update)
     await queryClient.cancelQueries({ queryKey: ["realEstateProperties"] });
 
     // Snapshot the previous value
-    const previousRealEstateProperties = queryClient.getQueryData<any>([
+    const previousRealEstateProperties = queryClient.getQueryData([
       "realEstateProperties",
     ]);
 
     // Optimistically update to the new value
     if (previousRealEstateProperties) {
-      queryClient.setQueryData<any>(["realEstateProperties"], (old: any) => [
+      queryClient.setQueryData(["realEstateProperties"], (old: any) => [
```

Comment:
Is there a way to type `old` in a better way than any?

Reply:
I tried to provide a better type here, but had some difficulty and decided to timebox the effort. If you like, I can add something to the backlog to revisit if there's something better we can do here.

Reply:
Discussed offline